.travis.yml

```diff
@@ -1,10 +1,19 @@
+os: 
+  - linux
+  - windows
+
```

Comment:
How about the Mac?

Reply:
Are we support our extensions for Mac?

Reply:
Design Editor need to support the Mac. WAPPL and TAUrine also support the MaC.
But I'm not sure travis also support the Mac os.
If the travis don't support the Mac os, you can skip it.